packages/usuario/src/domains/usuarios.js

```diff
@@ -0,0 +1,35 @@
+const { Op } = require('sequelize');
```

Comment:
I think in this case we shouldn't use de-structuring due to Op doesn't say anything about what this constant do